velox/functions/prestosql/registration/CMakeLists.txt

```diff
@@ -29,8 +29,11 @@ add_library(
   URLFunctionsRegistration.cpp
   RegistrationFunctions.cpp)
 
+set(SIMDJSON_LINK_LIBS ${SIMDJSON_LINK_LIBS})
```

Comment:
Where do we use SIMDJSON_LINK_LIBS variable?

Reply:
Sorry, this variable has not been used, deleted it.